notes/src/components/NotesList.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import List from "@mui/material/List"
+import Note from "./Note"
+import styles from "../pages/styled"
+import PropTypes from "prop-types"
+
+function Notes(props) {
+
+    return (
+        <div style={styles.AllNotes}>
+            <List>
+                {props.notes.map(note =>{
+                    return <Note note={note} clickNote={props.noteChosen}/>

```

Comment:
Only one return statement per function!
And here no need to use return at all. Use arrow function functionality please